### PR TITLE
Update run_alaska

### PR DESCRIPTION
**Pull request by bearecinos**
run_alaska.py with the new oggm version.

### CODE REVIEW

File: oggm/sandbox/run_alaska.py [hunk 1]
```diff
@@ -38,70 +38,67 @@
...
 # ---------------------------------
 # Note that to be exact, this procedure can only be applied if the run
 # parameters don't change between the calibration and the run.
 # After testing, it appears that changing the 'border' parameter won't affect
 # the results much (expectedly), so that it's ok to change it. All the rest
 # (e.g. smoothing, dx, prcp factor...) should imply a re-calibration
-mbf = 'https://www.dropbox.com/s/23a61yxwgpprs9q/ref_tstars_no_tidewater.csv?dl=1'
+mbf = 'https://github.com/OGGM/oggm-sample-data/blob/master/wgms/oggm_ref_tstars_rgi5_cru4.csv'
 mbf = utils.file_downloader(mbf)
 shutil.copyfile(mbf, os.path.join(WORKING_DIR, 'ref_tstars.csv'))
```
Comment (fmaussion) [inline oggm/sandbox/run_alaska.py]: This is not necessary anymore: the `distribute_tstars` task does it for you: https://github.com/OGGM/oggm/blob/master/oggm/core/climate.py#L1116
(that  is, if `cfg.PARAMS['run_mb_calibration']` is `False`, which is the default)

File: oggm/sandbox/run_alaska.py [hunk 2]
```diff
@@ -154,34 +175,37 @@
...
-    #Calculate glacier masks
     execute_entity_task(tasks.glacier_masks, gdirs)
 
-# The rest Prepro tasks
+# Pre-processing tasks
 task_list = [
     tasks.compute_centerlines,
     tasks.initialize_flowlines,
+    tasks.compute_downstream_line,
+    tasks.compute_downstream_bedshape,
```
Comment (fmaussion) [inline oggm/sandbox/run_alaska.py]: Why do you need these? They are only necessary for doing a run

File: oggm/sandbox/run_alaska.py [hunk 3]
```diff
@@ -220,18 +249,19 @@
     for gdir in gdirs:
         gdir.inversion_calving_rate = 0
 
+    cfg.PARAMS['correct_for_neg_flux'] = False
+    cfg.PARAMS['filter_for_neg_flux'] = False
```
Comment (fmaussion) [inline oggm/sandbox/run_alaska.py]: Yes, but be careful to use the very last oggm version here (after @TimoRoth last PR - without it these parameters might be ignored)

File: oggm/sandbox/run_alaska.py (hunk 1 above)
Comment (bearecinos) [reply oggm/sandbox/run_alaska.py]: So then I can just delete that whole thing?

File: oggm/sandbox/run_alaska.py (hunk 2 above)
Comment (bearecinos) [reply oggm/sandbox/run_alaska.py]: mm I wasn't sure really what to take out or leave ... :P

File: oggm/sandbox/run_alaska.py (hunk 3 above)
Comment (bearecinos) [reply oggm/sandbox/run_alaska.py]: I dont understand this comment? 

And also about this setting I keep think I will have to do separate runs. One for land terminating and one for tidewater glaciers. I think that there should be a distinction inside the function between non-tidewater and tidewater. So, we don't have to worry about making separate runs or making extra ifs in run_alaska.py

File: oggm/sandbox/run_alaska.py (hunk 1 above)
Comment (fmaussion) [reply oggm/sandbox/run_alaska.py]: Yes

File: oggm/sandbox/run_alaska.py (hunk 3 above)
Comment (fmaussion) [reply oggm/sandbox/run_alaska.py]: Disregard my comment - if you updated oggm after timo's last PR, you are good

**Comment by bearecinos** [comment]
Ok I alredy deleted the unnecessary lines ... is there anything else @fmaussion ? I think I am on the latest version so I should not worry about the cfg.PARAMS[] stuff

**Comment by bearecinos** [comment]
@fmaussion is there was something else wrong with this file? Is it possible to merge this?

**Comment by fmaussion** [comment]
Sorry I thought there still was something going on with the plots.

Just merged

**Comment by bearecinos** [comment]
well this script has no plotting ... and also I guess the plotting problem is not related to oggm

**Pull request merged**